style.css

```diff
@@ -4,15 +4,18 @@ html, body {
     height: 100%;
     font-family: sans-serif;
 }
-.contribute-bar {
+.contribute-text {
+    display: block;
+    margin-top: 20px;
+    margin-bottom: -35px;
```

Comment:
Well, it's just a trick. Definitely not the most spec-compliant style but it's supported everywhere.